addon/components/cf-field/input/table.js

```
@@ -107,6 +107,8 @@ export default Component.extend(ComponentQueryManager, {
 
       this.set("showModal", false);
     } catch (e) {
+      // eslint-disable-next-line no-console
+      console.error(e);
```

Leftover I guess?

---

No, this is actually intended to avoid swallowing exceptions.

---

We used this for debugging the `Array.flat`-IE11-affair..

---

Let's keep the error logs until we have fully deployed ember-caluma in all our (main) projects and have more time to harden and clean-up the code.

---

@anehx True, but I still like to have those statements around. Lets discuss in person (ETA in office 30min :wink: )